src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/BaseRecalibratorDataflow.java

```diff
@@ -167,49 +172,48 @@ protected static String pickOutputTablesPath(boolean remote, String stagingLocat
 
     /** reads local disks or GCS -> header, and PCollection */
     private PCollection<Read> ingestReadsAndGrabHeader(final Pipeline pipeline, List<String> filenames) throws IOException {
-
-        if (filenames.size() > 1) {
-            throw new UserException("Sorry, we only support a single input file for now.");
-        }
-        // TODO: support more than one input.
-        String beforePath = filenames.get(0);
-
-        // input reads
-        if (BucketUtils.isCloudStorageUrl(beforePath)) {
-            // set up ingestion on the cloud
-            // but read the header locally
-            GcsPath path = GcsPath.fromUri(beforePath);
-            InputStream inputstream = Channels.newInputStream(new GcsUtil.GcsUtilFactory().create(pipeline.getOptions())
-                    .open(path));
-            SamReader reader = SamReaderFactory.makeDefault().validationStringency(ValidationStringency.SILENT).open(SamInputResource.of(inputstream));
-            header = reader.getFileHeader();
-
-            final SAMSequenceDictionary sequenceDictionary = header.getSequenceDictionary();
-            final ReadFilter readFilter = BaseRecalibratorWorker.readFilter();
-            final List<SimpleInterval> intervals = BRAC.intervalArgumentCollection.intervalsSpecified() ? BRAC.intervalArgumentCollection.getIntervals(sequenceDictionary) :
-                    IntervalUtils.getAllIntervalsForReference(sequenceDictionary);
-            return new ReadsSource(beforePath, pipeline).getReadPCollection(intervals, ValidationStringency.SILENT)
-                    // keep only the ones BQSR's interested in.
-                    .apply(new ReadsFilter(readFilter, header));
-        } else {
-            // ingestion from local file
-            SamReader reader = SamReaderFactory.makeDefault().validationStringency(ValidationStringency.SILENT).open(new File(beforePath));
-            header = reader.getFileHeader();
-            List<Read> readLst = new ArrayList<>();
-            ReadFilter readFilter = BaseRecalibratorWorker.readFilter();
-            for (SAMRecord sr : reader) {
-                if (!readFilter.test(sr)) continue;
-                try {
-                    Read e = ReadConverter.makeRead(sr);
-                    readLst.add(e);
-                } catch (SAMException x) {
-                    logger.warn("Skipping read " + sr.getReadName() + " because we can't convert it.");
-                } catch (NullPointerException y) {
-                    logger.warn("Skipping read " + sr.getReadName() + " because we can't convert it. (null?)");
+            if (filenames.size() > 1) {
```

Comment:
Actually, could you revert the formatting change here, since it seems to be going away from our default style instead of towards it.